web/html/web-introduction/css-defined.md

```diff
@@ -32,7 +32,7 @@ CSS does the following well: ???
 CSS does not do the following well: ???
 
 * Responsive Layouts, Text styling, Animation creation, Container positioning
-* Media insertion, Link creation, Database Storage, Math Expression
+* Media insertion, Link creation, Database Storage
```

Comment:
If I'm understanding the term correctly, I think that CSS can probably do Media insertion too?

For example:

```css
body {
  background: url(sweettexture.jpg) blue;
}
```

Reply:
The point is not what it "can do", but rather what it "can do best"

Reply:
Will check this tomorrow

Reply:
I see.

How about we change the language to say what is it intended and not intended for?

The thing is, what CSS can/cannot do is time-sensitive, new features in the future could invalidate this statement. 

We should also make it sounds less official if possible ("Database Storage" is confusing to a beginner), especially since this is an insight in the early stages of the Web topic.

Here are some excerpts from our latest content guidelines draft:

> In legacy topics in particular, ensure we're communicating the "why", and the "how to get set up / get started" early on
> Don’t use words that your non-technical friend won't understand, like “compiled”, “pre-execution”, “stringified” -- unless they are absolutely necessary in which case you explain them
> Thing of mental models! Like in the first insight of Comp Sci. How can we make things as VISUAL as possible, especially early on?

Maybe there's something out of these that we can apply here?